secboot/secboot_sb.go

```diff
@@ -188,3 +194,67 @@ func UnlockEncryptedVolumeWithRecoveryKey(name, device string) error {
 
 	return nil
 }
+
+func ActivateVolumeWithKey(volumeName, sourceDevicePath string, key []byte, options *ActivateVolumeOptions) error {
+	return sb.ActivateVolumeWithKey(volumeName, sourceDevicePath, key, (*sb.ActivateVolumeOptions)(options))
+}
+
+func DeactivateVolume(volumeName string) error {
+	return sb.DeactivateVolume(volumeName)
+}
+
+func AddInstallationKeyOnExistingDisk(node string, newKey keys.EncryptionKey) error {
```

Comment:
maybe AddBootstrapKeyToFreshDisk or something along those lines would be a clearer name for this, given other naming decisions?